### DISCUSSION TITLE
removeAll when empty array passed for a collection using assign

### DISCUSSION THREAD

**Discussion by ghost** [mikro-orm/mikro-orm#6135] [Ideas]
Provide an option [or make it the default] to run a removeAll() when an empty array is passed in assign. 

`
// resets the addresses collection to a single item
wrap(user).assign({ addresses: [new Address(...)] });

// adds new address to the collection
wrap(user).assign({ addresses: new Address(...) });

/* NEW FEATURE */
// resets the addresses collection 
wrap(user).assign({ addresses: [] });
`

Thoughts? 

Thanks for all the hard work, awesome package :)

**Comment by B4nan** [comment]
this is how it works already, provide a repro if you think otherwise.

**Comment by ghost** [comment]
Indeed, it does. Thanks
